src/xrGame/ui/UIActorMenu.h

```diff
@@ -258,6 +258,7 @@ class CUIActorMenu : public CUIDialogWnd, public CUIWndCallback
     bool AllowItemDrops(EDDListType from, EDDListType to);
 
     bool OnItemDrop(CUICellItem* itm);
+    bool OnItemDropOnItem(EDDListType t_old, EDDListType t_new, CUIDragDropListEx* old_owner, CUIDragDropListEx* new_owner);
```

Comment:
Bad name. Nichego ne ponyatno...

Reply:
`DropItemOnAnotherItem` - tak norm?))

Reply:
What this function does? :eyes:

Reply:
> What this function does? eyes

This function calls when player drag and drop an item on another item in a slot in the inventory or weapon/outfit slot. The function call Lua function and then code on Lua side determinate whats doing with these items.